templates/header.hbs

```diff
@@ -0,0 +1,19 @@
+  <header class="header">
+
+    <nav class="header__container">
+
+      <a class="header__logo" href="#">
```

Comment:
Maybe is a good Idea to have an aria-label here. What you can try is included it and make the image decorative : alt="" role="presentation", so the screen reader will read the aria-label insted of the image and potentially avoid a11y problems.

Reply:
The logo is also a link to the homepage

Reply:
Correct but the link functionality is given by the anchor, no by the image, so will be good to have the aria-label